src/lib.rs

```diff
@@ -164,25 +169,45 @@ pub fn initialize_check_rotation(device_context: &DeviceContext) -> Result<InitA
         .block_on(initialize_check_rotation_async(device_context))
 }
 
+fn form_rotation_needed(
+    rotations_needed: EitherOrBoth<UserId, Vec1<GroupId>>,
+    ironoxide: IronOxide,
+) -> InitAndRotationCheck {
+    InitAndRotationCheck::RotationNeeded(
+        ironoxide,
+        PrivateKeyRotationCheckResult { rotations_needed },
+    )
+}
+
 async fn initialize_check_rotation_async(
```

Comment:
Now that this is an `async fn`, could probably remove the `_async` suffix

Reply:
Good point. We have some work to do to separate sync and async APIs. We can fix this up at that point.